workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.test.ts

```diff
@@ -32,6 +33,7 @@ describe('DynamoDBService', () => {
         pk: 'pk',
         sk: 'sk'
       };
+      base64DecodingOfPaginationToken = 'eyJwayI6InBrIiwic2siOiJzayJ9';
```

Comment:
why change the pagination token?

Reply:
Our APIs expects a base64 encoded string for the pagination token, not an object. So I updated the implementation and therefore this test needs to be updated as well.

Reply:
Ahh was not aware of that. Good catch then. Maybe just delete the object all together if its not needed anymore?

Reply:
So the object is used when we mock the DDB call, so it's still needed for that.